main.go

```diff
@@ -66,13 +66,13 @@ func main() {
 		log.Fatalf("Failed to initialize Kubernetes client: %v.", err)
 	}
 
-	awsSess, err := session.NewSession()
+	awsCfg, err := awsconfig.LoadDefaultConfig(context.TODO())
```

Comment:
Why `TODO()` and not something like `Background()`?

Reply:
Updated!